### PR TITLE
Move cloudbuild step of packaging an example into the runner

### PR DESCRIPTION
**Pull request by juliettepretot**
See the discussion in https://project-oak.slack.com/archives/CHE9E13C3/p1594984760219500 for background. 

# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: runner/src/main.rs [hunk 1]
```diff
@@ -449,6 +449,22 @@ fn run_ci() -> Step {
                     coverage: false,
                 },
             }),
+            // Package the Hello World application in a Docker image.
+            run_examples(&RunExamples {
```
Comment (rbehjati) [inline runner/src/main.rs:453]: Could this be a separate command that can be run individually? I think that would be helpful when working locally.

Comment (juliettepretot) [reply runner/src/main.rs:453]: If you wanted to run just this step, you can do so via: `./scripts/docker_run ./scripts/runner run-examples --example-name=hello_world --run-server=false --run-clients=false --build-docker`

Comment (juliettepretot) [reply runner/src/main.rs:453]: The code here basically provides the same arguments, just in the shape of a step inside of the `run-ci` task.

Comment (rbehjati) [reply runner/src/main.rs:453]: You are right. Thanks :)

**Pull request merged**